.eslintrc.json

```diff
@@ -54,7 +54,9 @@
         "leadingUnderscore": "allow"
       }
     ],
-    "@typescript-eslint/no-non-null-assertion": "off"
+    "@typescript-eslint/no-non-null-assertion": "off",
+    "@typescript-eslint/no-use-before-define": "error",
+    "no-use-before-define": "off"
```

Comment:
`no-use-before-defined` is turned off in favor of the TypeScript version.

Source: https://github.com/typescript-eslint/typescript-eslint/issues/2540#issuecomment-692866111